pkgs/development/coq-modules/hierarchy-builder/default.nix

```diff
@@ -5,22 +5,26 @@ with lib; mkCoqDerivation {
   owner = "math-comp";
   inherit version;
   defaultVersion = with versions; switch coq.coq-version [
-    { case = isGe "8.12";         out = "1.0.0"; }
+    { case = isGe "8.12";         out = "1.1.0"; }
     { case = range "8.11" "8.12"; out = "0.10.0"; }
   ] null;
+  release."1.1.0".sha256  = "sha256-spno5ty4kU4WWiOfzoqbXF8lWlNSlySWcRReR3zE/4Q=";
   release."1.0.0".sha256  = "0yykygs0z6fby6vkiaiv3azy1i9yx4rqg8xdlgkwnf2284hffzpp";
   release."0.10.0".sha256 = "1a3vry9nzavrlrdlq3cys3f8kpq3bz447q8c4c7lh2qal61wb32h";
   releaseRev = v: "v${v}";
 
   nativeBuildInputs = [ which ];
 
-  propagatedBuildInputs = [ coq-elpi ];
+  propagatedBuildInputs = [ coq-elpi coq.ocaml ];
```

Comment:
Why does OCaml need to be propagated?

Reply:
You are right, I do not think it does need to.